src/app/(homepage)/Demo.js

```diff
@@ -24,8 +25,14 @@ export default function Demo() {
   return (
     <Container sx={{ py: { xs: 6, md: 10 } }}>
       <DemoHeader sx={{ mx: "auto" }} />
-      <OosStepper sx={{ mt: 4, maxWidth: 700, mx: "auto" }} steps={steps} />
-      <Box component="img" src="/demo.png" sx={{ mt: { xs: 2, sm: 10 }, width: "100%" }} />
+      <OosStepper sx={{ mt: 4, maxWidth: 700, mx: "auto", mb: { xs: 2, sm: 10 } }} steps={steps} />
+        <NextImage 
+          src="/demo.png"
+          width={0}
```

Comment:
Did you read it?
![image](https://github.com/digital-brother/ooscca-fe/assets/43747573/db38d2df-8a01-44b4-83ac-b69fb02f860e)